test/oli/sections_test.exs

```diff
@@ -2145,4 +2154,65 @@ defmodule Oli.SectionsTest do
       refute Sections.get_latest_visited_page(section.slug, student.id)
     end
   end
+
+  describe "has_instructor_role?/2" do
+    setup do
+      user = insert(:user)
+      section = insert(:section)
+      {:ok, %{section: section, user: user}}
+    end
+
+    test "returns true when user is enrolled as content developer", %{
+      section: section,
+      user: user
+    } do
+      Sections.enroll(user.id, section.id, [ContextRoles.get_role(:context_content_developer)])
+
+      assert Sections.has_instructor_role?(user, section.slug)
+    end
+  end
+
+  describe "instructors_per_section/1" do
+    test "returns user name for section if user is enrolled as content developer" do
+      section = insert(:section)
+      user = insert(:user, name: "Content Dev")
+      Sections.enroll(user.id, section.id, [ContextRoles.get_role(:context_content_developer)])
+
+      result = Sections.instructors_per_section([section.id])
+      assert Map.has_key?(result, section.id)
+      assert Enum.any?(result[section.id], &(&1 == "Content Dev"))
+    end
+  end
+
+  describe "contains_instructor_role?/1" do
+    alias Lti_1p3.Roles.ContextRoles
```

Comment:
I think we should remove this line as this alias is already defined along with the other aliases.

Reply:
Good catch! It's removed now